setup.py

```diff
@@ -30,25 +31,28 @@ def local_open(fname):
 for directory in os.walk('example/templates'):
     example_dirs.append(directory[0][8:] + '/*.*')
 
+with local_open(os.path.join('pages', '__init__.py')) as f:
+    exec(f.read(), {}, pages)
```

Comment:
I suggest what we had. 
Did you have some issue with it? It should work - this is best practice for many packages.